CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ Toutes les modifications notables apportées au projet seront documentées dans
 
 Le format est basé sur [Tenez un Changelog](https://keepachangelog.com/en/1.0.0/), et ce projet adhère au [*Semantic Versioning*](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.9] - 2020-08-13
+
+### Ajouté
+- Import des agréments Pôle emploi de Juillet 2020
+- Ajout de liens vers YouTube et LinkedIn dans le footer
+- Affichage aux DE et prescripteurs de la lettre de motivation envoyée dans leurs candidatures
+- Ajout d'une possibilité de connexion automatique aux différents comptes de tests dans l'environnement de Démo
+
```

Comment:
J'aurais bien vu l'import de données ...

Reply:
Of course, j'ai zappé l'import PE lol